code/game/gamemodes/vampire/vampire.dm

```diff
@@ -78,6 +73,11 @@
 		for(var/datum/mind/vampire in vampires)
 			var/traitorwin = 1
 
+			var/datum/antagonist/vampire/V = vampire.has_antag_datum(/datum/antagonist/vampire)
+
+			if(!V)
+				continue
```

Comment:
If you're not accessing the contents of the vampire datum, an `is_vampire` check would work better here.